README.md

```diff
@@ -2,18 +2,20 @@
 
 ![screenshot](docs/screenshots/screenshot1.png)
 
-A web based Stars! clone.
+An open-source, web based Stars! clone/mobile port.
 
 [![Build Status](https://github.com/sirgwain/craig-stars/actions/workflows/build.yml/badge.svg)](https://github.com/sirgwain/craig-stars/actions/workflows/build.yml)
 
 ## About
 
-`craig-stars` is an interstellar, turn based strategy game and a member of the 4X genre (eXplore, eXpand, eXploit, eXterminate). The original [Stars!](https://wiki.starsautohost.org/) was written by Jeff Johnson and Jeff McBride with help from Jeffrey Krauss (collectively known as the Jeffs). `craig-stars` attempts to remain as close to the original as possible in both UI and gameplay.
+[Stars!](https://wikipedia.org/en/Stars!) is an interstellar, turn based strategy game and a member of the 4X genre (eXplore, eXpand, eXploit, eXterminate). The original Stars! game was released as shareware by Jeff Johnson and Jeff McBride with help from Jeffrey Krauss (collectively known as the Jeffs), with a retail version published in 1997. `craig-stars` attempts to remain as close to the original as possible in both UI and gameplay, while also adding functionality and allowing it to function on mobile devices.
```

Comment:
Not sure I like this statement in the README professing mobile support. Now people will _expect_ it to work! 😀

Reply:
I mean it _does_ work quite well tbf...